components/atoms/ToggleOption/toggle-option.tsx

```diff
@@ -13,7 +13,7 @@ const ToggleOption = ({ optionText, withIcon, checked, handleToggle }: ToogleOpt
       className="inline-flex cursor-pointer items-center gap-2 rounded-md px-2 py-0.5 border border-light-slate-6 bg-light-slate-1"
     >
       <ToggleSwitch size="sm" handleToggle={handleToggle} name={optionText} checked={checked} />
-      <span className="  text-sm text-light-slate-11">{optionText}</span>
+      <span className="text-sm  text-light-slate-11">{optionText}</span>
```

Comment:
nit: remove extra spaces

```suggestion
      <span className="text-sm text-light-slate-11">{optionText}</span>
```